test/claw_reference_test.cmake

```diff
@@ -349,17 +363,30 @@ function(claw_add_advanced_test)
     COMMENT "${CLAWFC} for CPU target on file ${ORIGINAL_FILE}"
   )
 
-  # Execute the CLAW Compiler for GPU target
+  # Execute the CLAW Compiler for ACC target
   add_custom_command(
-    OUTPUT  ${OUTPUT_FILE_GPU}
+    OUTPUT  ${OUTPUT_FILE_ACC}
     COMMAND touch ${ORIGINAL_FILE} # to force new compilation
     COMMAND ${CMAKE_COMMAND} -E env CLAW_TRANS_SET_PATH=${CLAW_TRANS_SET_PATH}
-      ${CLAWFC} ${claw_add_advanced_test_CLAW_FLAGS} --target=gpu
-      ${claw_add_advanced_test_CLAW_FLAGS_TARGET_GPU}
-      ${DEBUG_FLAG} -J ${XMOD_DIR} -o ${OUTPUT_FILE_GPU} ${ORIGINAL_FILE}
+      ${CLAWFC} ${claw_add_advanced_test_CLAW_FLAGS} --target=gpu --directive=openacc
+      ${claw_add_advanced_test_CLAW_FLAGS_TARGET_ACC}
+      ${DEBUG_FLAG} -J ${XMOD_DIR} -o ${OUTPUT_FILE_ACC} ${ORIGINAL_FILE}
     WORKING_DIRECTORY ${claw_add_advanced_test_WORKING_DIRECTORY}
     DEPENDS ${ORIGINAL_FILE}
-    COMMENT "${CLAWFC} for GPU target on file ${ORIGINAL_FILE}"
+    COMMENT "${CLAWFC} for ACC target on file ${ORIGINAL_FILE}"
```

Comment:
GPU target with ACC directive is more accruate

Reply:
ok, np